src/components/FlawAffects/__tests__/FlawAffects.spec.ts

```diff
@@ -233,6 +233,26 @@ describe('flawAffects', () => {
     expect(resolutionOptions.includes('DEFER')).toBe(false);
   });
 
+  osimFullFlawTest('Changing affect to not affected automatically set\'s empty resolution', async () => {
+    const affectsTableRows = subject.findAll('.affects-management table tbody tr');
+    const affectRowEditBtn = affectsTableRows[0].find('td:last-of-type button:first-of-type');
+    expect(affectRowEditBtn.exists()).toBe(true);
+    await affectRowEditBtn.trigger('click');
+
+    const affectsTableEditingRows = subject.findAll('.affects-management table tbody tr.editing');
+    const affectednessSelect = affectsTableEditingRows[0].find('td:nth-of-type(5) select');
+    await affectednessSelect.setValue('NOTAFFECTED');
+    const selectedAffectedness = affectednessSelect.find('option[selected]');
+    expect(selectedAffectedness.text()).toBe('NOTAFFECTED');
+
+    const affectRowCommitBtn = affectsTableEditingRows[0].find('td:last-of-type button:first-of-type');
+    expect(affectRowCommitBtn.exists()).toBe(true);
+    await affectRowCommitBtn.trigger('click');
+
+    const resolutionSelect = affectsTableRows[0].find('td:nth-of-type(6) span');
+    expect(resolutionSelect.text()).toBe('');
```

Comment:
Could you add another expect at the beginning for the resolution? How do we know that it wasn't always `''`?

Reply:
Makes sense, added.